pkg/api/v1/testkube/model_test_workflow_extended.go

```diff
@@ -115,3 +120,19 @@ func (w TestWorkflow) HasService(name string) bool {
 
 	return false
 }
+
+func (w *TestWorkflow) DeepCopy() *TestWorkflow {
+	if w == nil {
+		return nil
+	}
+	v, err := json.Marshal(w)
+	if err != nil {
+		panic(fmt.Sprintf("cannot deep copy testkube.TestWorkflow: %v", err))
```

Comment:
in Clone method marshaling errors are ignored, but panic in this method

Reply:
No problem, changed to ignore errors - this path would never be hit anyway.